src/query/executor/transform/lazy.go

```diff
@@ -32,6 +33,12 @@ type sinkNode struct {
 	block block.Block
 }
 
+// Params for sinkNode returns an params object for lazy sinks. It doesn't appear in queries (since sink nodes are
```

Comment:
Think this one may have fallen through the cracks (thanks github for collapsing comments for some reason 🙄 )